content/katas/compound-interest/index.md

```diff
@@ -1,47 +0,0 @@
----
-title: Compound interest
-pre: "<b>1. </b>"
-weight: 5
----
-
-## compound Interest
-
-**1.**
-Write a function that calculates compount interest. It should return a number.
-
-Eg if I have R100 in a bank account and I get 1% interest every month, I can see how much money I will have in my account at the end of each month like so:
-
-Try it with a loop. Try it with recursion.
-
-```
-// JavaScript
-
-function finalAmount(startAmount,interest,iterations){
-    // clever things
```

Comment:
Please remove these file changes from your branch, because they will, if merged, destroy the codebase.
use this command to remove any traces from your branch:

`git rm --cached name_of_file`

eg. 
`git rm --cached content/katas/compound-interest/index.md`

then `git push`

**OR**

Review your code and delete everything from Github :) 
you choose!